org.libvips.vipsdisp.json

```diff
@@ -0,0 +1,386 @@
+            "buildsystem" : "cmake-ninja",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/AcademySoftwareFoundation/Imath.git",
+                    "tag" : "v3.1.3",
+                    "commit" : "6dc0820482aeb24a22f520d7b165a410589648c3"
+                }
+            ]
+        },
+
+        {
+            "name" : "openexr",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/AcademySoftwareFoundation/openexr.git",
+                    "tag" : "v3.1.1",
+                    "commit" : "81e3d1beffb27448f6d610ed1b1cc1c962d51011"
+                }
+            ]
+        },
+
+        {
+            "name" : "pkg-config",
+            "builddir" : true,
+            "buildsystem" : "autotools",
+            "config-opts": [ 
+                "--with-pc-path=/usr/lib/x86_64-linux-gnu/pkgconfig" 
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://pkgconfig.freedesktop.org/releases/pkg-config-0.29.2.tar.gz",
+                    "sha256" : "6fc69c01688c9458a57eb9a1664c9aba372ccda420a02bf4429fe610e7e7d591"
+                }
+            ]
+        },
+
+        {
+            "name" : "autoconf",
+            "builddir" : true,
+            "buildsystem" : "autotools",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.gz",
+                    "sha256" : "954bd69b391edc12d6a4a51a2dd1476543da5c6bbf05a95b59dc0dd6fd4c2969"
+                }
+            ]
+        },
+
+        {
+            "name" : "automake",
+            "builddir" : true,
+            "buildsystem" : "autotools",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://ftp.gnu.org/gnu/automake/automake-1.14.1.tar.xz",
+                    "sha256" : "a9b4f04b8b69cac2e832a38a718943aa976dbdad0097211f8b3448afdacf0669"
+                }
+            ]
+        },
+
+        {
+            "name" : "openslide",
+            "builddir" : true,
+            "buildsystem" : "autotools",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/openslide/openslide/releases/download/v3.4.1/openslide-3.4.1.tar.gz",
+                    "sha256" : "fed08fab8a9b1ded95a34e196652291127ebe392c11f9bc13d26e760295a102d"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/openslide-3-fixes.patch"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/openslide-remove-configure.patch"
```

Comment:
this patch is not needed. There is the `rm-configure` option to remove the configure file.

Reply:
Oh neat, I missed that. Updated.